### ISSUE TITLE
scrollEnd Event returns no slidenumber

### ISSUE THREAD

**Issue by DotCoyote** [d4nyll/smartscroll#24]
Hi,

in your documentation, you write

> The scrollStart and scrollEnd listener will receive the slide number as its only argument; [...]

which would be very useful for me ;)
Unfortunately, your version doesn't return a slidenumber at scrollEnd, but "undefined"

I fixed that in a local copy:

smartscroll.js:174

``` javascript
windowTop = getWindowTop();
if (options.eventEmitter) {
  sectionIndexAtWindowMiddle = getSectionIndexAt(windowTop + $(window).height() / 2);
  options.eventEmitter.emitEvent('scrollEnd', [sectionIndexAtWindowMiddle]);
}
```

**Comment by d4nyll** [comment]
@CinKon it must have slipped my mind! Thanks for pointing it out! I hope to go through all the bugs in the issue tracker soon and update them all in one go.

**Comment by jmcguerreiro** [comment]
Just ran across this today, thanks for the fix @CinKon

**Comment by d4nyll** [comment]
Fixed in Release [2.5.3](https://github.com/d4nyll/smartscroll/releases/tag/2.5.3). Thank you @CinKon!
